org.eclipse.xtext.xbase.lib/build.gradle

```diff
@@ -5,3 +5,23 @@ description = 'The runtime library for Xbase languages.'
 dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
+
+task copyGwtSources(dependsOn: 'build') {
+	doLast {
+	
+		ant.copy(toDir: '../org.eclipse.xtext.xbase.lib.gwt/build/gwtSources', verbose: true) {
```

Comment:
yes but none of these seem to do a contains filtering, it all seems be name based filtering and actual content filtering with means of replacements